pkg/v1/providers/infrastructure-aws/v1.2.0/cconly/base.yaml

```diff
@@ -525,6 +554,49 @@ spec:
         path: "/spec/template/spec/rootVolume/size"
         valueFrom:
           variable: AWS_NODE_OS_DISK_SIZE_GIB
+
+  - name: AWS_NODE_LABELS
```

Comment:
nit: I don't see UPPER_SNAKE casing used in naming patches else where in the definition